src/strings/en-us.json

```diff
@@ -165,6 +165,8 @@
     "CustomDlnaProfilesHelp": "Create a custom profile to target a new device or override a system profile.",
     "DateAdded": "Date added",
     "DatePlayed": "Date played",
+    "DateAddedColon": "Date added:",
```

Comment:
It looks like you didn't use this string anywhere.

Reply:
Added for redundancy, in case Date Added was used anywhere we want a colon as well.

Reply:
Why not just put the colon in the in-code string where required? It seems weird to have a single string that exists just to "have a colon at the end".

Reply:
That works.